client-admin/src/app/shared/components/event-calendar/event-calendar.component.ts

```diff
@@ -207,6 +209,10 @@ export class EventCalendarComponent implements OnInit, OnChanges {
     return CzechHolidays(date.year).some(item => item.m === date.month && item.d === date.day);
   }
 
+  private isToday(date: DateTime): boolean {
+    return date.toMillis() === DateTime.now().set({ hour: 0, minute: 0, second: 0, millisecond: 0 }).toMillis();
```

Comment:
To je takový hack, co to nahradit tímhle?
```ts
date.hasSame(DateTime.now(), 'day');
```

Reply:
Může být, původně jsem zkoušel porovnávat přímo ty datetime objekty, ale nějak se neshodovaly - new datetime měl prostě něco navíc a tak to nešlo porovnávat... hasSame day mě jako trik nenapadl.